circuit_knitting/cutting/qpd/decompose.py

```diff
@@ -214,7 +224,13 @@ def _decompose_qpd_instructions(
         for data in inst.operation.definition.data:
             # Can ignore clbits here, as QPDGates don't use clbits directly
             assert data.clbits == ()
-            tmp_data.append(CircuitInstruction(data.operation, qubits=[qubits[0]]))
+            if basis_gate_set is None or data.operation.name in {"qpd_measure"}:
```

Comment:
Anything that comes out of `QPDBasis` can be translated except our measurement objects. Those never need to be translated